gems/aws-sdk-core/spec/aws/log/formatter_spec.rb

```diff
@@ -34,6 +34,10 @@ def format(pattern, options = {})
         end
 
         it 'provides a :request_params replacement' do
+          tmpfile = Tempfile.create
+          File.unlink(tmpfile.path) unless RUBY_DESCRIPTION.match?(/mswin|ming|cygwin/)
```

Comment:
Can we link a comment to the issue on why we should exclude these?

Reply:
done